rs/registry/transport/proto/ic_registry_transport/pb/v1/transport.proto

```diff
@@ -79,6 +79,20 @@ message RegistryValue {
   bool deletion_marker = 3;
 }
 
+message RegistryValueV2 {
+  uint64 version = 2;
+
+  oneof content {
+    bytes value = 1;
+    bool deletion_marker = 3;
+    LargeValueChunkKeys large_value_chunk_keys = 4;
+  }
```

Comment:
New oneof containing two old fields (and one new field, but it's not actually used).